packages/legacy/core/App/screens/AttemptLockout.tsx

```diff
@@ -109,32 +111,42 @@ const AttemptLockout: React.FC = () => {
   }, [dispatch, state.loginAttempt.loginAttempts])
 
   return (
-    <SafeAreaView style={styles.container}>
+    <SafeAreaView style={styles.container} edges={['bottom', 'left', 'right']}>
```

Comment:
The top edge being excluded here prevents a lot of unwanted space at the top of the screen